it/test/helpers/agent/AgentComponentSpecBase.scala

```diff
@@ -47,7 +47,7 @@ import uk.gov.hmrc.http.{HeaderCarrier, SessionId}
 
 import scala.concurrent.{ExecutionContext, Future}
 
-trait ComponentSpecBase extends TestSuite with CustomMatchers
+trait AgentComponentSpecBase extends TestSuite with CustomMatchers
```

Comment:
this is fine to do these changes (rename this triat), but again, these to be done agreed if we need these between both teams; as other team member would expect fine things were their are; this is quite hight rate of changes in the project and necessity of these are questionable

Reply:
clean up of the naming similar named files causing issues when juggling changes and investigating issues, and differentiating between the two component specbases, can differentiate between the two based on path but it simpler and better for general searching within the codebase.

Reply:
totally agree, its a good idea, but we if we can move this under different story, so to have as small PRs a possible; but again, check with your TL pls for other comments

Reply:
he's off for three weeks starting this week, prior to that his last comment was to add a test for checking if the audit triggers for showing error page scenario. He reviewed it prior and I told him about the build.sbt changes and logback changes so he should have seen them when I asked him for a review, Tom went on leave then Chandra after.

Reply:
I would suggest to stick to the smaller changes a possible and avoid changes in other areas, thats fine to create other stories along the way (tech debt etc) when you work on the story; we just need to estimate / refine these quickly; smaller changes reduce number of merge conflicts if any, simplify testing + there many other reasons; but again, its up to the PR reviewer to decide if approve these changes or not;